code/modules/antagonists/space_phoenix/space_phoenix.dm

```diff
@@ -6,7 +6,45 @@
 	assigned_by = ANTAGONIST_SOURCE_RANDOM_EVENT
 	objectives = list(/datum/objective/specialist/phoenix_collect_humans, /datum/objective/specialist/phoenix_collect_critters, /datum/objective/specialist/phoenix_permafrost_areas)
 	success_medal = "Territorial Defender"
+	/// How far from the map edge does our nest need to be
 	var/map_edge_margin = 35
+	/// The ability holder of this space phoenix
+	var/datum/abilityHolder/space_phoenix/ability_holder
+
+	give_equipment()
+		. = ..()
+		var/datum/abilityHolder/space_phoenix/A = src.owner.current.get_ability_holder(/datum/abilityHolder/space_phoenix)
+		if (!A)
+			src.ability_holder = src.owner.current.add_ability_holder(/datum/abilityHolder/space_phoenix)
+		else
+			src.ability_holder = A
+
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/sail)
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/thermal_shock)
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/ice_barrier)
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/glacier)
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/wind_chill)
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/touch_of_death)
+		src.ability_holder.addAbility(/datum/targetable/critter/space_phoenix/permafrost)
+
+		src.owner.current.setStatus("phoenix_empowered_feather", INFINITE_STATUS)
```

Comment:
This is a temporary buff, so think it would be better in New(), and no need to remove in remove_equipment()

Reply:
gotta work on how it's managed so it's not relying on a spawn in the future, probably, but will do this for now